sdk/servicebus/azure-servicebus/azure/servicebus/management/_utils.py

```diff
@@ -309,16 +309,14 @@ def serialize_rule_key_values(entry_ele, rule_descripiton):
 
 
 # Helper functions for common parameter validation errors in the client.
-def _validate_entity_name_type(entity_name, display_name="entity name"):
-    # type: (str, str) -> None
+def _validate_entity_name_type(entity_name: str, display_name: Optional[str] ="entity name"):
```

Comment:
it looks like this should always be string, but correct me if I'm wrong

```suggestion
def _validate_entity_name_type(entity_name: str, display_name: str ="entity name"):
```

Reply:
you're right